src/main/java/org/broadinstitute/hellbender/tools/exome/CreatePanelOfNormals.java

```diff
@@ -309,146 +296,18 @@ private OptionalInt calculatePreferredNumberOfEigenSamples() {
-    }
-
-    /**
-     * Calculate the number of eigen sample given the user preferences and the
-     * result of the log-normals SVD.
-     *
-     * @param requestedNumberOfEigenSamples the user requested eigen values (empty means the user didn't specify any in particular).
-     * @param numberOfCountColumns number of count columns in the original input.
-     * @param logNormalsSVD SVD results on the log-normal counts.
-     * @return always greater than 0.
-     */
-    @VisibleForTesting
-    static int determineNumberOfEigenSamples(final OptionalInt requestedNumberOfEigenSamples, final int numberOfCountColumns, final SVD logNormalsSVD, final Logger logger) {
-        final int numberOfEigenSamples;
-        if (requestedNumberOfEigenSamples.isPresent()) {
-            if (requestedNumberOfEigenSamples.getAsInt() > numberOfCountColumns) {
-                logger.warn(String.format("The number of requested eigen samples (%d) is larger than the available number of read count columns after filtering (%d), thus we will have to use the latter.", requestedNumberOfEigenSamples.getAsInt(), numberOfCountColumns));
-            }
-            numberOfEigenSamples = Math.min(requestedNumberOfEigenSamples.getAsInt(), numberOfCountColumns);
-        } else {
-            final double[] singularValues = logNormalsSVD.getSingularValues();
-            final double mean = MathUtils.mean(singularValues, 0, singularValues.length);
-            final double eigenValueCutoff = mean * JOLLIFES_RULE_MEAN_FACTOR; // Jollife's less strict version of Kaiser' Rule.
-            numberOfEigenSamples = (int) DoubleStream.of(singularValues).filter(v -> v > eigenValueCutoff).count();
-            logger.info(String.format("Jollife's rule produced %d eigen samples out of %d possibles for the reduced PoN", numberOfEigenSamples, numberOfCountColumns));
-        }
-        return numberOfEigenSamples;
-    }
-
-    /**
-     * Calculates the median of column medians and subtract it from all counts.
-     * @param readCounts the input counts to center.
-     * @return the median of medians that has been subtracted from all counts.
-     */
-    @VisibleForTesting
-    static double subtractBGSCenter(final ReadCountCollection readCounts, final Logger logger) {
-        final RealMatrix counts = readCounts.counts();
-        final int columnCount = counts.getColumnDimension();
-        final double[] columnMedians = new double[columnCount];
-        final double[][] data = counts.getData();
-        final Median medianCalculator = new Median();
-        final double[] columnData = new double[data.length];
-        for (int i = 0; i < columnCount; i++) {
-            for (int j = 0; j < data.length; j++) {
-                columnData[j] = data[j][i];
-            }
-            columnMedians[i] = medianCalculator.evaluate(columnData);
+    private double calculateExtremeColumnMedianCountsPercentileThreshold() {
```

Comment:
IMO it's preferable for method names to refer to targets and samples rather than rows and column -- the underlying matrix isn't part of the API.